src/components/pages/signUp.js

```diff
@@ -13,37 +14,46 @@ const SignUp = () => {
     firstName: "",
     lastName: "",
     email: "",
-    pass: "",
-    confirmPass: "",
+    password: "",
+    confirmpassword: "",
   });
 
   const [errors, setErrors] = useState({});
 
-  const { firstName, lastName, email, pass, confirmPass } = inputs;
+  let history = useHistory();
 
-  const handleChange = (e) => {
+  const { firstName, lastName, email, password, confirmpassword } = inputs;
+
+  const handleChange = e => {
     const { name, value } = e.target;
     setInputs((inputs) => ({ ...inputs, [name]: value }));
   };
 
-  const handleSubmit = (e) => {
+  const handleSubmit = e => {
     e.preventDefault();
 
+    axios
+      .post("http://localhost:3000/signup", inputs)
```

Comment:
no need to query another port. Just add proxy config to webpack.config.js
like `proxy: 'http://localhost:3000'`